app/commands/codemod/reviewpr.go

```diff
@@ -0,0 +1,74 @@
+// The review package contains helper methods to create review requests to be sent to the bot
+// endpoint layer, especially CLAIR.
+package codemod
+
+import (
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/juju/errors"
+)
+
+type PROpts struct {
+	Host     string
+	HostName string
+	Owner    string
+	Name     string
+	PRID     int
+}
+
+// ParsePR produces a set of PROpts to be sent to CLAIR in the bot endpoint layer
+func ParsePR(urlStr string) (*PROpts, error) {
+	// TODO: Try to parse urlStr as a request for each VCS host
```

Comment:
Make sure you add this TODO as an issue, milestone, or in your project management tool; eg. Trello.